core-android/src/main/java/org/hisp/dhis/client/sdk/android/api/persistence/PersistenceModuleImpl.java

```diff
@@ -317,6 +317,7 @@ public boolean deleteAllTables() {
                 enrollmentStore.deleteAll() &&
                 trackedEntityInstanceStore.deleteAll() &&
                 trackedEntityAttributeValueStore.deleteAll() &&
-                optionSetStore.deleteAll();
```

Comment:
@xurxodev explains me the user`s Attributes are downloaded during the loging, and his solution is valid.